src/exampleTypes/ResultTable/Header.js

```diff
@@ -27,6 +27,14 @@ const moveColumn = (
   })
 }
 
+const addColumn = (mutate, computeSideIndex, includes, triggerField, field) => {
+  let index = includes.indexOf(triggerField)
+  if (index >= 0) {
+    includes.splice(computeSideIndex(index), 0, field)
+    mutate({ include: includes })
```

Comment:
if you named `includes` consistently as `include`, you could use object shorthand syntax here